calendar_resource/models/calendar_event.py

```diff
@@ -64,38 +68,20 @@ def _format_datetime_interval_list(self, intervals):
                 'zallday': False,
                 'zduration': 24,
             }
-            datetimes += '\n%s\n' % self._get_display_time(**args)
-        return datetimes
+            datetimes.append(self._get_display_time(**args))
+        return '\n\n'.join(datetimes)
```

Comment:
The issue is the newline char itself, which differs between operating systems - https://en.wikipedia.org/wiki/Newline

On your Windows box, newline is `\r\n`